cibuild.sh

```diff
@@ -325,6 +325,18 @@ function c-cache {
   ln -sf `which ccache` $prebuilt/ccache/bin/avr-g++
 }
 
+function objcopy {
+
+  if ! type objcopy > /dev/null; then
+    case $os in
+      Darwin)
+        brew install binutils
+        add_path /usr/local/binutils/bin/
```

Comment:
For Darwin should we do the add_path prior to checking if we need to install?

Reply:
done